### ISSUE TITLE
callback for this.fs.copyTpl

### ISSUE THREAD

**Issue by johannesjo** [yeoman/generator#783]
Would be really nice to have some. Currently I'm trying to work around this with async's until. But this seems hacky.

**Comment by SBoudrias** [comment]
This method is synchronous. There's no need for a callback.

**Comment by johannesjo** [comment]
When I try to manipulate the file to be created directly afterwards, it is not available for some reason. I'll debug a little more, maybe this is some other issue.

**Comment by SBoudrias** [comment]
The files are all written to disk at the same time once a generator is done running. Before that they're all stored in memory to allow composition.

You can read http://yeoman.io/authoring/file-system.html
